### ISSUE TITLE
Is this still alive?

### ISSUE THREAD

**Issue by roddyyaga** [nojb/ocaml-imap#37]
Is this project still being worked on? I managed to hack together a version of the current master (see my fork) that more or less runs the minimutt example, but it still gets some errors. I'm happy to do some work on it if that would be useful.

**Comment by nojb** [comment]
No, the project is not actively maintained. Feel free to fork it and do something with it if you like :)
